docs/install_guides/installing-openvino-windows.md

```diff
@@ -84,7 +84,7 @@ The core components are now installed. Continue to the next section to configure
 You must update several environment variables before you can compile and run OpenVINO™ applications. Open the Command Prompt, and run the `setupvars.bat` batch file to temporarily set your environment variables:
 
 ```sh
-"<INSTALL_DIR>\openvino_2022\setupvars.bat"
+"<INSTALL_DIR>\setupvars.bat"
```

Comment:
@slyubimt could you confirm this comment is valid for windows?

Reply:
Yes, It would be nice to point the default installation path on Windows:
C:\Program Files (x86)\Intel\openvino_<version>
and symbolic link:
C:\Program Files (x86)\Intel\openvino_2022